docs/others/observer.rst

```diff
@@ -171,13 +171,19 @@ Unsubscribe from a subject
 
 .. code:: c
 
-    lv_observer_remove(observer)
+    lv_observer_remove(observer);
 
 To unsubscribe from a subject with all widgets you can use:
 
 .. code:: c
 
-    lv_subject_remove_obj(subject, obj)
+    lv_subject_remove_obj(subject, obj);
```

Comment:
Is this API still available?

`lv_obj_remove_from_subject` is bit of awkward. 

Is `lv_subject_remove_obj ` better?

Reply:
I've reviewed it and realized that `lv_subject_remove_all_obj` is not that useful so I removed it.

However in case of `lv_obj_remove_from_subject` we know the object, and we want to remove it from a secific or all subjects. Therefore I think it makes more sense to make it an `lv_obj` function.